overlay/usr/local/bin/turnkey-regen-ldap-certs

```diff
@@ -116,4 +116,26 @@ certtool --generate-certificate \
 
 chown -R openldap:openldap $TLS
 chmod 640 $cert_files
-systemctl restart slapd
+if systemd-detect-virt -c 2>&1>/dev/null; then
+    # workaround for systemctl stop not working for slapd on container builds
+    echo "WARN: using workaround for stopping slapd on container ..."
+    pid=$(pgrep slapd)
+    systemctl stop slapd
+    sleep 5
+    if ps -p $pid >/dev/null; then
+        kill $pid
+    fi
+    for _ in {0..10}; do
```

Comment:
Not sure if this line should be indented for a little more clarity but it was a little confusing for me to tell if it was part of the if loop or not. No big deal though, I don't think the indentations affect how the code runs 😊